### PR TITLE
Gab/view user

### PR DESCRIPTION
**Pull request by gidoo5331**
## PR Name or Description

View single-user details

## Overview

Added a modal for viewing single users and fixed the dropdown menu not closing.

## Detailed Technical Change

Added a new util to resolve DropdownMenu `pointer-events: none` persisting on the body after closing a dialog.
Added view modal.

## Testing Approach & Result

Screenshot from 2024-12-19 09-57-51
Screenshot from 2024-12-19 09-58-04

## Related Work

N/A

## Documentation

N/A

### CODE REVIEW

File: apps/admin/modules/user/view/index.tsx [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+      <Dialog open={opened} onOpenChange={() => setOpened(false)}>
+        <DialogOverlay className="bg-black/10" />
+        <DialogContent
+          className={`sm:max-w-[850px] bg-white dark:bg-black top-96`}
+        >
+          <DialogHeader className="flex-row items-center gap-3">
+            <Avatar className="w-20 h-16 rounded-md">
+              <AvatarImage src={data?.photo} alt="Profile picture" />
+              <AvatarFallback>
+                {_.upperCase(
```
Comment (arkhurst) [inline apps/admin/modules/user/view/index.tsx:47]: I think we can move this into a re-usable `util` or `component`. Unless we would not re-use this anyhere

**Review by arkhurst — COMMENTED** [review]
Code looks good generally, however I added 

-  A suggestion
- And a question

**Comment by arkhurst** [comment]
> ## PR Name or Description
> View single-user details
> 
> ## Overview
> Added a modal for viewing single users and fixed the dropdown menu not closing.
> 
> ## Detailed Technical Change
> Added a new util to resolve DropdownMenu `pointer-events: none` persisting on the body after closing a dialog. Added view modal.
> 
> ## Testing Approach & Result
> Screenshot from 2024-12-19 09-57-51 Screenshot from 2024-12-19 09-58-04
> 
> ## Related Work
> N/A
> 
> ## Documentation
> N/A

Any specific reason why we're passing `Details` to the modal header

File: apps/admin/modules/user/view/index.tsx (hunk 1 above)
Comment (gidoo5331) [reply apps/admin/modules/user/view/index.tsx:47]: I think we might.

Comment (Bendomey) [reply apps/admin/modules/user/view/index.tsx:47]: ala, my lodash people  🤣🤣🤣🤣

I forget that package sef. It's uneccessarily huge for me (I'm not saying remove it 🤣)

Comment (arkhurst) [reply apps/admin/modules/user/view/index.tsx:47]: You're not a good human being

**Review by arkhurst — APPROVED** [review]

**Pull request merged**